drivers/bluetooth/hci/userchan.c

```diff
@@ -181,7 +181,7 @@ static void rx_thread(void *p1, void *p2, void *p3)
 
 	LOG_DBG("started");
 
-	uint16_t frame_size = 0;
+	int frame_size = 0;
```

Comment:
Wouldn't `ssize_t` be a better type, considering the usage (e.g. doing operations together with other `ssize_t` variables like `len`)?

That said, there's `int32_t decoded_len` later in the function which might be better updated to `ssize_t` too.

Reply:
changed :)